shell/docker-builder.sh

```diff
@@ -27,15 +27,13 @@ source "${LIB_DIR}/ssh-auth.sh"
 secrets=("--secret" "id=npmtoken,env=NPM_TOKEN")
 args=("--ssh" "default" "--progress=plain" "--file" "deployments/${appName}/Dockerfile" "--build-arg" "VERSION=${VERSION}")
 
-# Build a quick native image on PRs and load it into docker cache
-# for security scanning
-if [[ -z $CIRCLE_TAG ]]; then
-  info "Building Docker Image (test)"
-  docker buildx build "${args[@]}" "${secrets[@]}" -t "${appName}" --load .
-
-  info "🔐 Scanning docker image for vulnerabilities"
-  "${TWIST_SCAN_DIR}/twist-scan.sh" "${appName}" || echo "Warning: Failed to scan image"
-fi
+# Build a quick native image and load it into docker cache for security scanning
+# Scan reports for release images are also uploaded to OpsLevel (test image reports only available on PR runs as artifacts).
+info "Building Docker Image (for scanning)"
+docker buildx build "${args[@]}" "${secrets[@]}" -t "${appName}" --load .
+
+info "🔐 Scanning docker image for vulnerabilities"
```

Comment:
The dev images are pushed from this repo https://github.com/getoutreach/devenv-snapshots ... perhaps I am not understanding this clearly, but should this change not happen there?

Reply:
Oh I see, this is for any image that we build .... What happens with helmcharts ( I dont believe those use the docker build commands) etc? I would imagine this should also apply to those right?

Additionally should this scan only happen on CI? This would mean that everytime someone does a local build/bootstrap we would end up scanning for vulnerabilities right?

Reply:
This is only used for CI, local builds use `make docker-build` in the Dockerfile. Helm charts are different, and we don't build those.

Reply:
yep, this is CI only. The only diff here is that scanning will now happen for both private PRs and for release PRs (OpsLevel will be reported on release image scans only, this has not changed)